Library/Formula/postgresql.rb

```diff
@@ -89,10 +128,16 @@ def caveats; <<-EOS.undent
 
     To migrate existing data from a previous major version (pre-9.4) of PostgreSQL, see:
       http://www.postgresql.org/docs/9.4/static/upgrading.html
+
+    Your existing data from the last major version, such as 9.3, has been preserved in
+    #{var}/postgres-9.3 (for example). There is a new #{var}/postgres-#{version_suffix} directory
+    for this version's data. Migrate data as desired.
```

Comment:
Might want to output this more specifically above rather than in the caveats on what was actually done and to what directories.